addons/account/models/account_move.py

```diff
@@ -1339,7 +1339,8 @@ def _compute_invoice_taxes_by_group(self):
             done_taxes = set()
             for line in tax_lines:
                 res.setdefault(line.tax_line_id.tax_group_id, {'base': 0.0, 'amount': 0.0})
-                res[line.tax_line_id.tax_group_id]['amount'] += line.price_subtotal
+                tax_balance_multiplicator = -1 if move.is_inbound(True) else 1
+                res[line.tax_line_id.tax_group_id]['amount'] += tax_balance_multiplicator * line.balance
```

Comment:
`line.amount_currency if line.currency_id else line.balance`

Reply:
There is a reference to price_subtotal left in the line 1359

Reply:
@smetl Should be ok now :)